src/app/garfile/amend/post.controller.js

```diff
@@ -0,0 +1,21 @@
+const logger = require('../../../common/utils/logger')(__filename);
+const CookieModel = require('../../../common/models/Cookie.class');
+const garApi = require('../../../common/services/garApi');
+const emailService = require('../../../common/services/sendEmail');
+const config = require('../../../common/config');
+
+module.exports = (req, res) => {
+  logger.debug('In garfile / amend post controller');
```

Comment:
There is no way to link a user with doing the amending of a GAR in the log e.g. no `userId` with log.
* If we are looking at the logs, it will be hard to know which user amended what GAR, which makes debugging tricker.

Reply:
updated log is as follows:

> In garfile / amend post controller - User: 9702bd59-9246-4e3b-9fd1-34904c206b79 editing GAR: d0aa5f84-84d3-4b92-91e5-45eea901ab32

Reply:
Cool, thanks.